droid-hal-device.inc

```diff
@@ -776,7 +776,8 @@ _remove_cruft   "/fstab.*"          \
                 "/charger"          \
                 "/res"              \
                 "/data"             \
-                "/odm/*"
+                "/odm/*"            \
+                "/sbin/mkdosfs"
```

Comment:
Would it make sense to exclude the whole path?

Reply:
Not sure if we can exclude everything from sbin, maybe leave it for later cleanup.